Firestore/core/src/firebase/firestore/remote/datastore.h

```diff
@@ -62,6 +73,10 @@ class Datastore {
   static GrpcStream::MetadataT ExtractWhitelistedHeaders(
       const GrpcStream::MetadataT& headers);
 
+  util::AsyncQueue* worker_queue_ = nullptr;
+  auth::CredentialsProvider* credentials_ = nullptr;
+  FSTSerializerBeta* serializer_;
```

Comment:
This could also be nullptr for consistency?

Reply:
Done.